pkg/apis/internal.acorn.io/v1/imageselector.go

```diff
@@ -0,0 +1,25 @@
+package v1
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+type SignedBy struct {
```

Comment:
General comment for all the structs here, but given that this feature has a lot of moving parts can we leave GoDoc comments explaining each of these? That should make our future-selves quite happy.